suitcase.py

```diff
@@ -20,44 +17,49 @@ def export(headers, filename):
     """
     with h5py.File(filename) as f:
         for header in headers:
-            assert isinstance(header, MutableMapping)
-            header = copy.deepcopy(header)
+            header = dict(header)
             try:
-                descriptors = header.pop('event_descriptors')
+                descriptors = header.pop('descriptors')
             except KeyError:
                 warnings.warn("Header with uid {header.uid} contains no "
                               "data.".format(header), UserWarning)
                 continue
-            top_group_name = repr(header).replace(' ', '_')[1:-1]
+            top_group_name = header['start']['uid']
             group = f.create_group(top_group_name)
             _safe_attrs_assignment(group, header)
             for i, descriptor in enumerate(descriptors):
-                desc_group = group.create_group('Event_Stream_{0}'.format(i))
+                # make sure it's a dictionary and trim any spurious keys
+                descriptor = dict(descriptor)
+                descriptor.pop('_name', None)
+
+                desc_group = group.create_group(descriptor['uid'])
+
                 data_keys = descriptor.pop('data_keys')
                 _safe_attrs_assignment(desc_group, descriptor)
+
                 events = list(find_events(descriptor=descriptor))
                 event_times = [e['time'] for e in events]
-                desc_group.create_dataset('event_times', event_times)
+                desc_group.create_dataset('time', data=event_times)
```

Comment:
To match the name in the event documents so we don't have to keep track of any mappings.